x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authc/AuthenticationContext.java

```diff
@@ -48,17 +48,6 @@ public Subject getEffectiveSubject() {
         return effectiveSubject;
     }
 
-    public Authentication toAuthentication() {
-        return new Authentication(
-            effectiveSubject.getUser(),
-            authenticatingSubject.getRealm(),
-            effectiveSubject.getRealm(),
-            version,
-            type,
-            authenticatingSubject.getMetadata()
-        );
-    }
-
```

Comment:
I was aiming to track all the`RealmRef` constructions as well as the `Authentication` where those end up being part of. I am not quite there yet in this draft. I would like to get to a point where `Authentication`s are only constructed in a couple of places and in those it is clear which are part of a domain and what that domain is.

Since this method is not used anywhere, I thought it was just easier to remove than to reason about.
I would still lean on removing it.

Reply:
I have no objection since it's not used anyway. Thanks